File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/ThreadInfo.java
package work.socialhub.kslack.entity;

public class ThreadInfo {

    private boolean complete;
    private Integer count;

    public boolean isComplete() {
        return this.complete;
    }

    public Integer getCount() {
        return this.count;
    }

    public void setComplete(boolean complete) {
        this.complete = complete;
    }

    public void setCount(Integer count) {
        this.count = count;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/ThreadInfo.kt
package work.socialhub.kslack.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlin.js.JsExport

/**
 * - https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-model/src/main/java/com/slack/api/model/ThreadInfo.java#L6
 */
@JsExport
@Serializable
class ThreadInfo {

    @SerialName("complete")
    var isComplete: Boolean = false

    var count: Int? = null
}
